src/main/java/org/skypro/examgen/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,34 @@
+package org.skypro.examgen.controller;
+
+import org.skypro.examgen.model.Question;
+import org.skypro.examgen.service.JavaQuestionService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+    private final JavaQuestionService questionService;
```

Comment:
не забываем про полиморфизм
```suggestion
    private final QuestionService questionService;
```